src/components/CoinListHeader/index.js

```diff
@@ -1,24 +1,91 @@
-import { Row } from "antd";
-import { ColumnHeader } from "./CoinHeader.styles";
+import React from "react";
+import { CaretSymbol } from "../CaretSymbol";
+import { Row, Col } from "antd";
+import { RiHeartFill } from "react-icons/ri";
+import { ColumnHeader, FavIcon } from "./CoinHeader.styles";
 
-export const CoinListHeader = (props) => {
+export const CoinListHeader = ({ order, handleSort, sortBy }, ...rest) => {
   return (
     <Row>
-      <ColumnHeader span={1}>
-        <div>Fav</div>
+      <ColumnHeader lg={{ span: 1 }}>
+        <FavIcon>
+          <RiHeartFill size="1rem" color="#ff7b7b" />
+        </FavIcon>
       </ColumnHeader>
-      <ColumnHeader span={1}>
-        <div>Rank</div>
+      <ColumnHeader
+        lg={{ span: 1 }}
+        onClick={() => handleSort("market_cap_rank")}
+      >
+        <div>
+          #{sortBy !== "market_cap_rank" ? "" : <CaretSymbol value={order} />}
+        </div>
       </ColumnHeader>
-
-      <ColumnHeader span={5}>
-        <div>Name</div>
+      <ColumnHeader lg={{ span: 4 }} onClick={() => handleSort("name")}>
+        <div>
+          Name
+          {sortBy !== "name" ? "" : <CaretSymbol value={order} />}
+        </div>
       </ColumnHeader>
-      <ColumnHeader span={2}>
-        <div>Price</div>
+      <ColumnHeader
+        lg={{ span: 2 }}
+        onClick={() => handleSort("current_price")}
+      >
+        <div>
+          Price
+          {sortBy !== "current_price" ? "" : <CaretSymbol value={order} />}
+        </div>
       </ColumnHeader>
-      <ColumnHeader span={2}>
-        <div>24h %</div>
+
+      <Col lg={{ span: 5 }}>
+        <Row>
```

Comment:
This Row element seems very repetitive, could you create a function and pass arguments to it so that it returns a ColumnHeader dynamic element.? Then call the function below each time with the arguments that you need.